src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 3 && days < 6) {
+    return 40 * days - 20;
+  }
+
+  if (days === 6) {
+    return 40 * days - 20;
+  }
+
+  if (days >= 7) {
+    return 40 * days - 50;
```

Comment:
Same comment as before about hard-coded values. It's better to use a constant for `7` and `50`.